src/pages/en/guides/publish-to-npm.md

```diff
@@ -6,37 +6,40 @@ description: Learn how to publish Astro components to NPM
 
 Building a new Astro component? **Publish it to [npm!](https://npmjs.com/)**
 
-Publishing a component is a great way to reuse work across your team, your company, or the entire world. Astro components can be published to and installed from npm, just like any other JavaScript package.
+Publishing an Astro component is easy and a great way to reuse work across your projects, with others, and with the wider Astro-community at large. Astro components can be published directly to and installed from NPM, just like any other JavaScript package.
 
-**Astro’s ability to publish and reuse popular components is one of it’s most powerful features!**
+Looking for inspiration? Check out some of [our favorite themes & components](/en/themes) from the Astro community. You can also [search npm](https://www.npmjs.com/search?q=keywords:astro-component) to see the entire public catalog.
 
-Even if you don't plan on publishing your components online, the patterns outlined below can help any developer design reusable components in isolation from their custom website or business logic.
+## Quick Start
 
-Looking for inspiration? Check out some of [our favorite themes & components][/themes] from the Astro community. You can also [search npm](https://www.npmjs.com/search?q=keywords:astro-component) to see the entire public catalog.
+```bash
+npm init astro myComponents -- --template component
+```
 
 ## Creating a package
 
-> Before diving in, it will help have a basic understanding of:
+> Before diving in, it will help to have a basic understanding of:
 >
 > - [Node Modules](https://docs.npmjs.com/creating-node-js-modules)
 > - [JSON Manifest (`package.json`)](https://docs.npmjs.com/creating-a-package-json-file)
 > - [Workspaces](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#workspaces)
 
-To create a new package, we recommend developing with **workspaces**. This will allow you to develop your component alongside a working copy of Astro.
 
-```
-my-project/
-   ├─ demo/
-      └─ ... for testing and demonstration
-   ├─ package.json
-   └─ packages/
-      └─ my-component/
-         ├─ index.js
-         ├─ package.json
-         └─ ... additional files used by the package
+To create a new package we strongly recommend configuring your development enviroment to use **workspaces** within your project. This will allow you to develop your component alongside a working copy of Astro.
+
+```tree
+myComponents/
+├─ demo/
```

Comment:
Spotting insignificant details is all I’m good for 😭 

```suggestion
└─ demo/
```

Reply:
Because I don't know what the actual structure is, or how separate demo is.... just checking that it's not this:


myComponents/
├─ demo/
 |   └─ ... for testing and demonstration
├─ package.json

Reply:
Oh yeah! That's right.